spec/classes/mongos_spec.rb

```diff
@@ -12,6 +12,16 @@
           '/etc/mongodb-shard.conf'
         end
       end
+      
+      describe 'with specific bind_ip values' do
```

Comment:
Would you mind moving this below the `context 'with defaults'` block? I think it's good to always first list tests with defaults, then variations on that.